tests/conftest.py

```diff
@@ -22,8 +22,7 @@
 from taipy.config import Config
 from taipy.config.common.frequency import Frequency
 from taipy.config.common.scope import Scope
-from taipy.core import Cycle, Job, Pipeline, Scenario, Task
-from taipy.core import DataNodeId, JobId
+from taipy.core import Cycle, DataNodeId, Job, JobId, Pipeline, Scenario, Task
```

Comment:
I would keep the 2-line import (separate Id import from entity import).

But it just conftest so it's not a big deal.